Makefile

```diff
@@ -25,10 +25,10 @@ profile:
 	@cp vimrc ~/.vimrc
 
 k8s:
-	curl -LO https://storage.googleapis.com/kubernetes-release/release/v1.16.5/bin/linux/amd64/kubectl
+	curl -LO https://storage.googleapis.com/kubernetes-release/release/v1.17.0/bin/linux/amd64/kubectl
```

Comment:
let's just go all the way to 1.20 since we know we'll be getting there eventually.  could you update this and test it out?